script/zonemaster_backend_rpcapi.psgi

```diff
@@ -45,15 +45,20 @@ Log::Any::Adapter->set(
                 'Screen',
                 min_level => get_loglevel(),
                 stderr    => 1,
+                newline   => 1,
                 callbacks => sub {
                     my %args = @_;
-                    $args{message} = sprintf "%s [%d] %s - %s\n", strftime( "%FT%TZ", gmtime ), $PID, uc $args{level}, $args{message};
+                    $args{message} = sprintf "%s [%d] %s - %s", strftime( "%FT%TZ", gmtime ), $PID, uc $args{level}, $args{message};
                 },
             ],
         ]
     ),
 );
 
+$SIG{__WARN__} = sub {
+    $log->warning(map { my $m = $_; $m =~ s/\n/ /g; $m =~ s/^\s+|\s+$//g; $m } @_);
```

Comment:
I'm not sure if you're aware this can be done without a temporary variable:
```perl
$log->warning( map s/^\s+|\s+$//gr, map s/\n/ /gr, @_ );
```

Reply:
This looks cleaner indeed, thanks